en/02_Developer_Guides/00_Model/11_Scaffolding.md

```diff
@@ -515,7 +586,7 @@ class Player extends DataObject
 Use a single search field that matches on multiple database fields with `'match_any'`. This also supports specifying a `FormField` and a filter, though it is not necessary to do so.
 
 > [!CAUTION]
-> If you don't specify a `FormField`, you must use the name of a real database field as the array key instead of a custom name so that a default field class can be determined.
+> If you don't specify `field` or `dataType`, you must use the name of a real database field as the array key instead of a custom name so that a default field class can be determined.
```

Comment:
`dataType` is new and was intended for use with the range stuff, but it also works here so makes sense to document this option.